signed/verifiers.go

```diff
@@ -116,6 +126,29 @@ func (v RSAPSSVerifier) Verify(key data.Key, sig []byte, msg []byte) error {
 	return verifyPSS(pubKey, digest[:], sig)
 }
 
+// RSAPKCS1v15SVerifier checks RSA PKCS1v15 signatures
+type RSAPKCS1v15Verifier struct{}
```

Comment:
For future reference I made the call of including PKCSv15 in here. The idea is that we don't know what kind of hardware we're going to deal with in the future, and it's better safe than sorry.

I hope we don't regret this decision. Everyone should be strongly encouraged to use PSS.